var/spack/repos/builtin/packages/albany/package.py

```diff
@@ -0,0 +1,128 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Albany(CMakePackage):
+    """Albany is an implicit, unstructured grid, finite element code for the
+       solution and analysis of multiphysics problems.  The Albany repository
+       on the GitHub site contains hundreds of regression tests and examples
+       that demonstrate the code's capabilities on a wide variety of problems
+       including fluid mechanics, solid mechanics (elasticity and plasticity),
+       ice-sheet flow, quantum device modeling, and many other applications."""
+
+    # Add a proper url for your package's homepage here.
+    homepage = "http://gahansen.github.io/Albany"
+    url      = "https://github.com/gahansen/Albany/tarball/master"
+
+    maintainers = ['gahanse']
+
+    # Add proper versions and checksums here.
+    version('master', git='https://github.com/gahansen/Albany.git', branch='master')
+
+    variant('install',          default=True,
+            description='Enable Install')
```

Comment:
what does this mean?

Reply:
There is currently an explicit install option (on or off) in Albany's build process. I can eliminate this if I clean that option up on the Albany side. I'll do that and eliminate this option.

Reply:
On further inspection, some Albany users need this option. I'll explicitly set it in the update instead of having it as a variant.